### DISCUSSION TITLE
What is your speed?

### DISCUSSION THREAD

**Discussion by jfranecki** [volotat/SD-CN-Animation#125] [General]
I am getting ~2.25 it/s on vid2vid and text2vid(with input video) with these settings. 

GPU: 4090
CPU: Ryzen 9 5950X
RAM: 32gb
I am using vladmandic automatic
[settings.txt](https://github.com/volotat/SD-CN-Animation/files/11539807/2.txt)

Update: I received this speed with a 1080x1920 resolution. After reducing that to 528x944, I now receive ~7.75 it/s and the output images work better with this smaller resolution.

**Comment by conor-is-my-name** [comment]
It seems to me that the switching back and forth between controlnet is what's really slowing everything down

**Reply by jfranecki** [reply]
Yeah I don't think the preprocessor can be cached since it needs to create a controlnet image for each frame. But I know what you mean, I see this before each frame is generated.
```
Loading model from cache: control_v11f1e_sd15_tile [a371b31b]
Loading preprocessor: tile_resample
preprocessor resolution = 512
Calling preprocessor tile_resample outside of cache.
```

**Comment by jfranecki** [comment]
Switched from Vlad to Auto1111 and it doubled my speed! Now I'm getting an average of 5.6 it/s

**Reply by peteTater** [reply]
While using a pruned model, at 1024x576, and the standard unchanged settings on the txt2vid tab, with no Lora, TI, Hypernetworks, etc in the prompts, or any video or image input, I have found that it generally takes just under 20 minutes to complete a 120 frame sequence on my 3060. That's roughly the speed each time, whether using --xformers or --opt-sdp-no-mem-attention, and without the token merging extension which I'm yet to test out. I think it was taking closer to 27/28 mins before using either of those commandline_args. If you have the time at some point to run a test for comparison, may I ask how quickly a sequence of 120 frames, with standard txt2vid settings and 1024x576 dimensions, completes processing for you using the 4090 under similar conditions? Really curious to know how great a speed up it could offer for SD-CN.
